### PR TITLE
Issue #553 - Reposition login buttons and links on desktop mode

### PR DESCRIPTION
**Pull request by andrewroblesdev**
The login buttons were made wider and the links were spaced out to match the alignment of the rest of the content on the login page. See attached screenshot to result.
Screen Shot 2021-03-06 at 5 07 10 AM

### CODE REVIEW

File: kofta/src/app/components/Button.tsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ export const Button: React.FC<
             variant === "small"
               ? "py-1 px-2 w-max"
               : variant === "slim"
-              ? "max-w-md ml-auto mr-auto py-2.5 px-12"
+              ? "max-w-2xl ml-auto mr-auto py-2.5 px-12"
```
Comment (benawad) [inline kofta/src/app/components/Button.tsx]: I like that the buttons are not full width on desktop atm

File: kofta/src/app/components/Footer.tsx [hunk 2]
```diff
@@ -11,7 +11,7 @@ export const Footer: React.FC<FooterProps> = ({ isLogin }) => {
 	const { t } = useTypeSafeTranslation();
 
 	return (
-		<div className={`justify-around flex text-center`}>
+		<div className={`justify-between flex text-center`}>
```
Comment (benawad) [inline kofta/src/app/components/Footer.tsx:14]: will this smash the footer against the side on mobile?

File: kofta/src/app/components/Button.tsx (hunk 1 above)
Comment (andrewroblesdev) [reply kofta/src/app/components/Button.tsx]: Sounds good, I will remove this change from the pull request

File: kofta/src/app/components/Footer.tsx (hunk 2 above)
Comment (andrewroblesdev) [reply kofta/src/app/components/Footer.tsx:14]: No I don't think so. Looks almost identical to how it was before on mobile: below is the resulting screenshot. If you don't like it, I can revert this too no problem.
Screen Shot 2021-03-06 at 5 54 02 AM

**Comment by andrewroblesdev** [comment]
Update: the width of the buttons has been reverted. Attached is the screenshot result of the most recent state of the pull request:
Screen Shot 2021-03-06 at 6 02 07 AM

**Pull request merged**